src/containers/Trip/Trip.js

```diff
@@ -70,17 +70,22 @@ class Trip extends Component {
         const { city, country, departure_date, return_date } = this.state.trip
         return(
             <div className='container mt-5'>
-                <div className='row'>
-                    <h1>{ city }, { country }</h1>
-                    <div className='col-1'></div>
-                    <div className='' style={{}}>
-                        <p style={{ margin:'0' }}><span>{ departure_date.format('L') }</span> - <span>{ return_date.format('L') }</span></p>
-                        <p style={{ textAlign:'center', margin:'0' }}>{ moment().endOf(departure_date).to(return_date) }</p>
+                <div className='row justify-content-between'>
+                    <div className='col-lg-4' style={{}}>
+                        <div>
+                            <h5>Trip Details</h5>
+                            <h1 style={{fontSize:'3.5rem'}}>{ city }, { country }</h1>
```

Comment:
> `style={{fontSize:'3.5rem'}}`

Is there a reason you're putting these styles inline? It's generally better to use classes in your `.css` files for reusability and clarity.